### ISSUE TITLE
Error when using model installed from disk

### ISSUE THREAD

**Issue by ErikTromp** [explosion/spacy-transformers#139]
I installed a custom model from disk using examples/init_model.py (successfully). Now when I try to load it, I get:

```
>>> import spacy
>>> nlp = spacy.load('bert-base-dutch-cased/bertje-base')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\__init__.py", line 30, in load
    return util.load_model(name, **overrides)
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\util.py", line 166, in load_model
    return load_model_from_path(Path(name), **overrides)
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\util.py", line 211, in load_model_from_path
    return nlp.from_disk(model_path)
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\language.py", line 941, in from_disk
    util.from_disk(path, deserializers, exclude)
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\util.py", line 654, in from_disk
    reader(path / key)
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\language.py", line 936, in <lambda>
    p, exclude=["vocab"]
  File "pipes.pyx", line 228, in spacy.pipeline.pipes.Pipe.from_disk
  File "C:\Users\me\Anaconda3\lib\site-packages\spacy\util.py", line 654, in from_disk
    reader(path / key)
  File "pipes.pyx", line 219, in spacy.pipeline.pipes.Pipe.from_disk.load_model
  File "D:\spacy-transformers\spacy_transformers\_tokenizers.py", line 60, in from_bytes
    setattr(self, field, msg[field])
KeyError: 'init_kwargs'
```

**Comment by adrianeboyd** [comment]
I think this comes from having a version of `transformers` that's too recent.

`spacy-transformers==0.5.1` only supports `transformers==2.0.0`. I was able to load a newly created `bertje-base` model created with the same script using these exact versions.

There's not a new `spacy-tranformers` release yet, but there are updates to `master` that should address this problem and hopefully make it compatible with `transformers==2.5.0`. If you want to try installing `spacy-transformers` from source, it should be relatively easy to test if you'd like.

**Comment by ErikTromp** [comment]
Well I realized I installed the spacy model in the same folder as the disk-based BERT model. Installing it in a different folder fixed it, for me at least.

**Comment by ErikTromp** [comment]
Actually I'd have to reopen this issue, I can _load_ the model fine now but using it still causes errors:

```
>>> import spacy
>>> nlp = spacy.load('bertje-base')
>>> doc = nlp('dit is een goed test bericht')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Erik\Anaconda3\lib\site-packages\spacy\language.py", line 435, in __call__
    doc = proc(doc, **component_cfg.get(name, {}))
  File "C:\Users\Erik\Anaconda3\lib\site-packages\spacy_transformers\pipeline\wordpiecer.py", line 50, in __call__
    pieces = self.predict([doc])
  File "C:\Users\Erik\Anaconda3\lib\site-packages\spacy_transformers\pipeline\wordpiecer.py", line 93, in predict
    seg_words = self.model.tokenize(segment.text)
  File "C:\Users\Erik\Anaconda3\lib\site-packages\transformers\tokenization_utils.py", line 764, in tokenize
    if self.init_kwargs.get("do_lower_case", False):
AttributeError: 'SerializableBertTokenizer' object has no attribute 'init_kwargs'
```

**Comment by ErikTromp** [comment]
I also tried installing from source but that gives me the following errors

```
Extracting spacy_transformers-0.5.1-py3.6.egg to c:\users\erik\anaconda3\lib\site-packages
  File "c:\users\erik\anaconda3\lib\site-packages\spacy_transformers-0.5.1-py3.6.egg\spacy_transformers\pipeline\ner.py", line 54
    }`
     ^
SyntaxError: invalid syntax

spacy-transformers 0.5.1 is already the active version in easy-install.pth

Installed c:\users\erik\anaconda3\lib\site-packages\spacy_transformers-0.5.1-py3.6.egg
Processing dependencies for spacy-transformers==0.5.1
error: srsly 0.0.7 is installed but srsly<1.1.0,>=0.1.0 is required by {'spacy'}
```

**Comment by adrianeboyd** [comment]
Oh, I missed a second spot where the requirements need to be updated. Can you try updating `setup.cfg` to also require `transformers>=2.5.0,<2.6.0` and then try again? Double-check that the right version of `transformers` is installed afterwards and then recreate the model with the `init_model` script.

Also, sometimes packages don't get cleaned up well if you switch between package and source installations. You might need to uninstall the existing `spacy-transformers` a couple times to clean up older versions (and maybe even double-check in `venv/lib/python3.x/site-packages` to be sure there aren't any lingering directories).

If not, try a brand new environment, install `torch` however you would otherwise, and then install from source. The backtick is definitely a typo/bug, but `python setup.py install` and `pip install .` should ignore it, so you could try:

```
cd spacy-transformers
pip install -U -r requirements.txt
pip install .
```

**Comment by ErikTromp** [comment]
Yep, modifying setup.cfg and reinstalling did the trick

**Comment by MarveM** [comment]
Hi, 

I also installed the model from disk, and am getting a similar error when trying:

nlp = spacy.load('en_trf_bertbaseuncased_lg')

...
c:\users\miniconda3\envs\spacy_transformers\lib\site-packages\spacy_transformers\_tokenizers.py in from_bytes(self, bytes_data, exclude, **kwargs)
     58         msg = srsly.msgpack_loads(bytes_data)
     59         for field in self.serialization_fields:
---> 60             setattr(self, field, msg[field])
     61         self.finish_deserializing()
     62         return self

KeyError: 'init_kwargs'


Any ideas what could be causing this? I re-created my virtual-env from scratch.

spacy == 2.2.4
spacy-transformers   == 0.6.0
transformers   == 2.5.1


Many thanks in advance for any pointers!!!

**Comment by adrianeboyd** [comment]
Sorry, please downgrade to `0.5.2` for now. `spacy-transformers` `v0.6.0` was accidentally released too soon. It was yanked from pypi after about a day, but if it's in your cache `pip` may still find it, so specify the version with `pip install spacy-transformers==0.5.2`.

**Comment by MarveM** [comment]
Thank you so much for your quick response, that worked like a charm :).

**Comment by cherry979988** [comment]
Thank you! I encountered the same problem and solved it by rolling back to 0.5.2.
